cmd/notary/integration_test.go

```diff
@@ -2240,8 +2258,14 @@ func TestClientKeyImport(t *testing.T) {
 	// if there is hardware available, root will only be on hardware, and not
 	// on disk
 	assertNumKeys(t, tempDir, 2, 4, !rootOnHardware())
-	_, err = os.Open(filepath.Join(tempDir, "private", notary.NonRootKeysSubdir, newKeyID+".key"))
+	_, err = os.Open(filepath.Join(tempDir, notary.PrivDir, newKeyID+".key"))
+	require.NoError(t, err)
+	_, err = os.Open(filepath.Join(tempDir, notary.PrivDir, newKeyID+".key"))
```

Comment:
ah my bad, totally slipped through- I'll remove the duplicates in the next push with addressing further comments